specification/inference/put_custom/PutCustomRequest.ts

```diff
@@ -30,7 +30,46 @@ import { InferenceChunkingSettings } from '@inference/_types/Services'
 /**
  * Create a custom inference endpoint.
  *
- * You can create an inference endpoint to perform an inference task with a custom model that supports the HTTP format.
+ * The custom service gives more control over how to interact with external inference services that aren't explicitly supported through dedicated integrations.
+ * The custom service gives you the ability to define the headers, url, query parameters, request body, and secrets.
+ * The custom service supports the template replacement functionality, which enables you to define a template that can be replaced with the contents of a value that defines that key.
```

Comment:
> can be replaced with the contents of a value that defines that key.

Maybe this would be a bit clearer: `can be replaced with the value associated with that key`?